server/src/infra/graphql/query.rs

```diff
@@ -296,23 +297,23 @@ impl<Handler: BackendHandler> User<Handler> {
         self.attributes
             .iter()
             .find(|a| a.attribute.name.as_str() == "first_name")
-            .map(|a| a.attribute.value.unwrap())
+            .map(|a| a.attribute.value.as_str().unwrap_or(""))
```

Comment:
`unwrap_or_default`

Reply:
Done, did the same on the line below, and for last_name as well.